TheWarWithin/MageArcane.lua

```diff
@@ -2211,4 +2211,4 @@ spec:RegisterSetting( "cancel_pom", false, {
6JJbIqo0MJsnnFRobG79U7UMUuxqp1FuemCg6JAqXQquuYdXIPWQyivZO1TErxqGol6VALXnwMPwv74EdDvhgzdfDsBLqBXauUfy0rnJX0ygGO)gcmWdLj(7RRHkvk4hhDKLzfP2tLjWIN3zvN6bKoTftYAX5yaXmvwBbfYFlXQzUXUSIc1y(BVUN1m5n9VKjylfeMrSpQVgQJpb2rx5LLGbKwVIC5pJoGDxld4(Z4TV2ZW88)CzugQBtRaGxzbK6ojJaSgMqC(tQ(W)BeA3o7haKJPjW6rE8Z0av)zu1qDpQ2icgYWjFX5e8wQQH2y9Z0dcLXHdOqwTAJHgAIoFNQF(QffnNhOYe3EaMwHx(OVcV6rFfE9J(kqUL0pYlXuyj(7jqWoulexK6BvqfVbt))ikPgAxO)5FbrxvGGScIYjRrZKBqiW6TtfIwPJpBrPgMSw9cPY0ASxjBTageOpClWBE0waT(s4DN5dI3epXtWVT(wuRt3VNEvUZ(pTAfEN99zqgDhMpTYewJHD7KiZF0DdyEfUFYCdZ79uM)4mRpo6Ng0NeoykLjvZrw1LoLYXgD7uUmyURuAFvxs6l8V6M3EyEs6d6vzLY7ONZ69uqzi8YDEw7uu9GiM0LqRczAl97EAdCNfyvF49hQbw8YkPRScU1WS(bAc7LA6JYpHh067x(c2CZVebvFq3OQldXiSkjlvRfXO0Jlj3SGrmO1lNExNgCsiT89Y3mkA3YNOZry1hE6FdsecpKpS2piN6IWWJSe9X8oRVeR19Co5Ci863MJ(UY0YqR3GaNbdnDZbU5g5ABQERbwC6n3y92cC2KtfNEXBiaFU1D7aKM4w1SeN1(Wle7B)Aociv0yBYyutazt1MhY(zVpKPP(CxwiA8keCZnwV(acp2WvhqCcKU2acpWWvgqyeMX)RNhi20(6hHy()Iwtn3xrTVwZNlMgzTWRWFjyOIFNoe)9MVlhI)k9JAbHsmWWgmS9ryGA6dBr6dSClsDCU5g68O(b2qKAu(4wGK1q9NTYaRhEYa5Y2E2YPoIltZxccYk0UC9K1zGYDbe2bdno2HsNeYIxp(MBA1OWlMbEaEIrqQilY48Bs64iVX4FTg(d2gJ)fv4py7l5V6b)HyZ90N(uSpIVmfVZa)4xGhL4fN36RSqtBf82YIYdjydlfHNrv((0sS738jqdP9ie7OIAE5pGlsdBmISCUHSvJCWD04qKJ6dGsTRmN5NBOrhDg2TiJOhVc3nOt)PUTyT4ruuGdS8YdhaqoafAGoUBtRhH7ItB3ht8zNsWk8q9H0qurN)r7WYg8MB0XoV5gzL5PGY8t6(QYRzTzitKKG2m50tpopewxGkLA0qXpfglNpEG0IC205JjHqtZZhOYpmVxnao6oSPTbDDX8BnHzgyLKMQY3QcZaZjp8wsdgG)HeqBmhCu6ZmPLAnNHftgBK(KTQdsDnqyqIz5xmsOBewoPBAT)SdnF4hiWc70uRgyNApUyXTzlZLZMDAl3MYK3bVa3pxggMKdUSsWp4P3pQZ(QTl0lBBgZUj0FF6J5IbU9dJd8GvQiLcc4(SscGlyUDONHirxrJgiXL)d(xBdPmsu(qmq318JJejmfWnD)2Qf)Be9dab1Kqf9)qJTSX3Z8qZoegNb1USBD4d9GKy3Bzrk5EmBcBqPPSxuOSOhtq8P6BcO)J8lsBto2CLqGN1cwuBFTAcAYyXdgsNsLFU9hc5flhFcanqOngi)IZanj6IPzBVuuZgBfNUPgwyXCzVTs8buumkp0hiRXNAbU2au68C22UnMsSchUOYXcyEMC6i2xZ1Ltovp7VMXvxze9nkhwVaGU)M4BVR19rA2wd1LZIuPHjnw)Klvhi7iogYAkPVBMZGUH(3mCXYqQzFv3jqkkgcaHARUq4fc9ZZYBD78mst38GMKkDWd8tK27b(ZAo5mdCsrEw)YoAHIjoLDl(r5975mXHMKXlN7Gjj3qtQbYasJ3cECZaZ9VN0(tZC530jRSeIvSOA7KttxToyy3xqeNgdZ)WDxKSlUKAR4UctO3lVLMD2AMxth7mq)zt6yLMLDsz0HefCy33MO(SwCh79txQhtydU2HAovoj3SsTS5cMvkPeAKrjE)IgyX2TjLErh6oo01stpe7uNOVMNnqbXG8DgIAo9T)T)PpskXygeUHslJoy4WBxCY6dtqBhM681owdrs9n9v1XQqQveblEewoJX519SP9rxq0Q1sqY(mv87VJP4P6DoB6eL0NqDhh0K1d0Th4UoBYynNvTWTXrXlHWnXzj8MYfjeMlTjd84f8H6L6)iVJkgv5EGGCPrmIrigOpViQDZeJeLoJ8UUmjlwYcuAm8MY5SUWyDw41vrhsw5urvi8wzJQF)d0IbgM2epv9siI4(wYw6aBoSZEqVqmg3dM8VQ)2YSCk6QORBidoM7g0kYg)3f31LUyOIGxgQf9IfOiSOLTVMllN91hLI99zpYfOdnVyBEfLvpzvK8r(79iSgtEzJzL4hQvmUaekzlEucrb4eweftsw8ioX7HFpg)leuVEml2CYhEksnr9cebvr3b)EbUrlqdkadQ)iL5OBh(afDByFYcuFlo0JWGo0gXtiY0ChT7MVoHfNReHJV8pSb3EklFVFx(zGuN0rtinl5mmOJ4DTAkSHN(8Ht)UHgpK(2ht5IzN64WR4INyhWAlh22FUTKc(2g0FDyu1Evva9)za4BhE5f095zCWTGEmMR9ztuzBtOX2FG(ykArS3zaPbA9AnyOUA0OlqmzNdzrrlY13KFme)lv)g9Xq8wzoA8tKiJRR85r0PdlvJ1mRjUPw5OG5GGobP8sgEPfZCKRNH4N5qoww1Pv1TtxOORF92Fidnx3qhve0F7(jk0IQI9c)r98m0IWUMZPrCpti3eD3ifZvpCgWCVpONTcEWSII1kN1o40mAFzCN)WaArQWRBMyPZZAU)vMj0ftAtOZT5QqjLadUswVYRSyFA26vR4)3E53H)3EzYvDF9)(p]] )
```

Comment:
If you updated this export string, you have to increase the version number to current date.

Reply:
Wouldn't be my pr if I didn't forget something, made new commit